tests/unit/moduleapi/testrdb.tcl

```diff
@@ -183,46 +181,46 @@ tags "modules" {
                     }
                     $master exec
 
-                    if {$testType == "Delayed" || $testType == "Aborted"} {
-                        # Set master with a slow rdb generation, so that we can easily intercept loading
-                        # 10ms per key, with 500 keys is 5 seconds
-                        $master config set rdb-key-save-delay 10000
-
-                        # Wait for the replica to start reading the rdb
-                        wait_for_condition 100 100 {
-                            [s -1 async_loading] eq 1
-                        } else {
-                            fail "Replica didn't get into async_loading mode"
-                        }
-                    }
-
                     switch $testType {
                         "Aborted" {
+                            # Set master with a slow rdb generation, so that we can easily intercept loading
+                            # 10ms per key, with 500 keys is 5 seconds
+                            $master config set rdb-key-save-delay 10000
+
+                            test {Diskless load swapdb RedisModuleEvent_ReplAsyncLoad handling: during loading, can keep module variable same as before} {
+                                # Wait for the replica to start reading the rdb and module for acknowledgement
```

Comment:
let's also update the comment so state that we wanna abort only after the temp db was populated by REDISMODULE_AUX_BEFORE_RDB